tamr_unify_client/models/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,75 @@
+from tamr_unify_client.models.base_collection import BaseCollection
+from tamr_unify_client.models.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection(BaseCollection):
+    """Collection of :class'tamr_unify_client.models.project.attribute_mapping.AttributeMapping'
+    :param client: Client for API call delegation.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param resource_json: JSON data representing this resource
+    :param api_path: API path used to access this collection.
+    E.g. ``"projects/4/attributeMappings/19054-12"``
+    :type api_path: str
+    """
+
+    def by_relative_id(self, relative_id):
+        """Retrieve an attribute mapping by relative ID.
+       :param relative_id: The relative ID.
+       :type relative_id: str
+       :returns: The specified attribute mapping.
+       :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+       """
+        return super().by_relative_id(AttributeMapping, relative_id)
```

Comment:
`BaseCollection`'s `by_relative_id` works by calling `get` for `id`, so here something like `projects/4/attributeMappings/19054-12`. However, there is no `GET` endpoint for a single mapping, so this will always result in an error. You should override this function by getting all mappings for the project and finding the one with the correct ID

Reply:
Because of this, you may also want to consider making `AttributeMapping` not a `BaseResource`, since they don't have their own API endpoint.